Package.swift

```diff
@@ -51,6 +51,8 @@ var targets: [PackageDescription.Target] = [
             dependencies: ["NIO", "NIOHTTP1", "CNIOSHA1"]),
     .target(name: "NIOWebSocketServer",
             dependencies: ["NIO", "NIOHTTP1", "NIOWebSocket"]),
+    .target(name: "NIOHTTP1HelloWorldServer",
+            dependencies: ["NIO", "NIOHTTP1", "NIOConcurrencyHelpers"]),
```

Comment:
Any reason this isn't just part of `NIOHTTP1Server`?

Reply:
it should not be part of the pr at all... included by mistake.